src/main/java/edu/harvard/iq/dataverse/trsa/registry/util/JsfUtil.java

```diff
@@ -0,0 +1,69 @@
+package edu.harvard.iq.dataverse.trsa.registry.util;
+
+import java.util.List;
+import javax.faces.application.FacesMessage;
+import javax.faces.component.UIComponent;
+import javax.faces.context.FacesContext;
+import javax.faces.convert.Converter;
+import javax.faces.model.SelectItem;
+
+public class JsfUtil {
```

Comment:
Is this name too generic? Should it be TrsaJsfUtil since it's in the trsa package?